cmd/describe/upgrade/cmd.go

```diff
@@ -0,0 +1,78 @@
+/*
+Copyright (c) 2022 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package upgrade
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"github.com/openshift/rosa/pkg/ocm"
+	"github.com/openshift/rosa/pkg/rosa"
+)
+
+var Cmd = &cobra.Command{
+	Use:     "upgrade",
+	Aliases: []string{"appliance", "upgrade"},
+	Short:   "Show details of an upgrade",
+	Long:    "Show details of an upgrade",
+	Example: `  # Describe an upgrade-policy"
+  rosa describe upgrade`,
+	Run:    run,
+	Hidden: true,
+}
+
+func init() {
+	ocm.AddClusterFlag(Cmd)
+}
+
+func run(cmd *cobra.Command, argv []string) {
+	r := rosa.NewRuntime().WithOCM()
+	defer r.Cleanup()
+	cluster := r.FetchCluster()
+
+	// Try to find the cluster:
+	clusterID, _ := cluster.GetID()
+	r.Reporter.Debugf("Loading upgrade with id %q", clusterID)
+	upgrades, err := r.OCMClient.GetUpgradePolicies(clusterID)
+	if err != nil {
+		r.Reporter.Errorf("Failed to get upgrade with cluster id %q: %v", clusterID, err)
+		os.Exit(1)
+	}
+	if len(upgrades) < 1 {
+		r.Reporter.Errorf("No available upgrades for cluster id %q", clusterID)
+		os.Exit(1)
+	}
+
+	for _, upgrade := range upgrades {
+		fmt.Printf(`
+                %-28s%s
+		%-28s%s
+		%-28s%s
+		%-28s%s
+		%-28s%s
+		%-28s%s
+`,
+			"Id:", upgrade.ID(),
+			"Href:", upgrade.HREF(),
+			"Upgrade type:", upgrade.UpgradeType(),
+			"Cluster ID:", upgrade.ClusterID(),
+			"Schedule At:", upgrade.Schedule(),
+			"Next run:", upgrade.NextRun())
```

Comment:
Please make the `r` in `run` upper case to align with other commands where 2 letters appears in first-letter-capital.